code/ui/manager/src/components/preview/FramesRenderer.tsx

```diff
@@ -117,19 +119,22 @@ export const FramesRenderer: FC<FramesRendererProps> = ({
           return null;
         }}
       </Consumer>
-      {Object.entries(frames).map(([id, src]) => (
-        <Fragment key={id}>
-          <IFrame
-            active={id === active}
-            key={refs[id] ? refs[id].url : id}
-            id={id}
-            title={id}
-            src={src}
-            allowFullScreen
-            scale={scale}
-          />
-        </Fragment>
-      ))}
+      {Object.entries(frames).map(([id, src]) => {
+        const [key] = refs[id] ? refs[id].url.split('?') : [id];
```

Comment:
FTR the `id` here doesn't match the `ref` id ("id = `storybook-ref-${r.id}`") so this line of code is actually pointless/broken.